gradle-plugin/src/main/java/com/dpforge/manifestguard/ManifestGuardPlugin.kt

```diff
@@ -1,11 +1,13 @@
 package com.dpforge.manifestguard
 
 import com.android.build.api.artifact.SingleArtifact
-import com.android.build.api.variant.AndroidComponentsExtension
+import com.android.build.api.variant.ApplicationAndroidComponentsExtension
+import com.android.build.api.variant.Variant
 import com.android.build.gradle.AppPlugin
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 
+@ExperimentalStdlibApi
```

Comment:
@dsvoronin can we do something with the fact that we are unable to force a newer version of the Kotlin for the Gradle plugin?